internal/config/config.go

```diff
@@ -93,6 +97,9 @@ func LoadConfig(filePath string, fileName string) (*Config, error) {
 	v.SetDefault("nodeIP", "node")
 	v.SetDefault("httpServerPort", 0)
 
```

Comment:
why keeping this NL?

Reply:
No reason actually, I can get rid of it :)